app/upgrades/v2/constants.go

```diff
@@ -0,0 +1,5 @@
+package v2
+
+const UpgradeName = "v2"
+const UpgradeHeight = 56300 // (20 * 60 * 60) / 6 + 44300
+const UpgradeInfo = `'{"binaries":{"darwin/arm64":"https://github.com/tharsis/evmos/releases/download/v2.0.0/evmos_2.0.0_Darwin_arm64.tar.gz","darwin/x86_64":"https://github.com/tharsis/evmos/releases/download/v2.0.0/evmos_2.0.0_Darwin_x86_64.tar.gz","linux/arm64":"https://github.com/tharsis/evmos/releases/download/v2.0.0/evmos_2.0.0_Linux_arm64.tar.gz","linux/x86_64":"https://github.com/tharsis/evmos/releases/download/v2.0.0/evmos_2.0.0_Linux_x86_64.tar.gz","windows/x86_64":"https://github.com/tharsis/evmos/releases/download/v2.0.0/evmos_2.0.0_Windows_x86_64.zip"}}'`
```

Comment:
We need a checksum if we want to have these usable, otherwise cosmovisor may reject them with v1.0.0 by default:
https://github.com/cosmos/cosmos-sdk/blob/master/cosmovisor/README.md#auto-download